modules/adding-node-iso-configs.adoc

```diff
@@ -68,6 +68,14 @@ For more information, see "Root device hints" in the "Setting up the environment
 The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
 |A dictionary of host network configuration objects.
 
+|cpuArchitecture
+|Specifies the architecture of the nodes you are adding.
+|String
```

Comment:
Should this be replaced with the actual supported values, since there are probably only a few of them?

Reply:
We're currently working to have a larger set of supported architectures, not sure if it's worth provide an explicit list now that should be re-updated in a while.
Moreoever, it's important to highligh somehow the fact that this field is _optional_, ie it allows the user to override the default value only if/when required